tests/integration/test_arctic.py

```diff
@@ -190,3 +194,83 @@ def test_lib_rename_namespace(arctic):
 def test_lib_type(arctic):
     arctic.initialize_library('test')
     assert(arctic.get_library_type('test') == VERSION_STORE)
+
+
+MY_ARCTIC = None  # module-level Arctic singleton
+
+def f(library_name, total_writes):
+    my_pid = os.getpid()
+    data = [str(my_pid)] * 100
+    while True:
+        try:
+            vstore = MY_ARCTIC[library_name]  # wait for parent to initialize
+            break
+        except LibraryNotFoundException:
+            pass
+        time.sleep(random() * 0.2)
+    for i in range(total_writes):
+        if i % 20 == 0:  # add some randomisation, make sure that processes are multiplexed across time
+            time.sleep(random())
+        key = "{}_{}".format(my_pid, i)
+        vstore.write(key, data + [key])
+    for i in range(total_writes):
+        key = "{}_{}".format(my_pid, i)
+        assert vstore.read(key).data == data + [key]
+    print "\nDone pid={}".format(my_pid)
```

Comment:
dont think we need the 'print' do we? if we do, make it a function please

Reply:
removed the print statement